### PR TITLE
`LogNormalize` transform

### PR DESCRIPTION
**Pull request by ordabayevy**
This PR add `LogNormalize` transform described in #11

### CODE REVIEW

File: scvid/transforms/transforms.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import torch
+from torch import nn
+
+
+class LogNormalize(nn.Module):
```
Comment (mbabadi) [inline scvid/transforms/transforms.py:5]: Let's rename this to something more appropriate, like `ZScoreLogNormalize`.

File: scvid/transforms/transforms.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    """
+    LogNormalize gene counts with mean, standard deviation, and pseudo-count.
```
Comment (mbabadi) [inline scvid/transforms/transforms.py:7]: I belive I was sloppy with me description.  In the following expression:
$log(1 + C \times x_{ng} / \sum_g x_{ng})$

The pseudo-count is 1, and C, is fixed target total count (not a pseudo-count). Let's do the following:

- Rename `C` to `target_count`
- Make scaling by `std_g` optional via a `scale` boolean flag.
- If applicable, add `eps: float = 1e-6` (a new __init__ arg) to `std_g` in the denominator to obviate division by zero.
-

File: scvid/transforms/transforms.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+    Args:
+        mean_g: Means for each gene.
+        std_g: Standard deviations for each gene.
+        C: Gene epxression pseudo-count.
+    """
+
+    def __init__(self, mean_g: torch.Tensor, std_g: torch.Tensor, C: int = 10_000):
```
Comment (mbabadi) [inline scvid/transforms/transforms.py:15]: (see above)

**Review by mbabadi — CHANGES_REQUESTED** [review]

File: scvid/transforms/transforms.py (hunk 1 above)
Comment (mbabadi) [reply scvid/transforms/transforms.py:5]: or better yet,`ZScoreLog1pNormalize`.

File: scvid/transforms/transforms.py (hunk 2 above)
Comment (ordabayevy) [reply scvid/transforms/transforms.py:7]: Or maybe have `std_g` as `Optional[torch.Tensor] = None` and not apply it if it is None?

**Comment by ordabayevy** [comment]
@mbabadi ready for a review!

**Review by mbabadi — APPROVED** [review]

**Comment by mbabadi** [comment]
@ordabayevy oh wait

**Comment by mbabadi** [comment]
@ordabayevy now that you're at it, how about also safeguarding dividing by library size (+ eps)? we don't want a NaN if a user gives us an empty cell :D also, I am leading toward adding `perform_scaling` bool.  It will be a stronger commitment from the user. Always a good thing!

**Comment by mbabadi** [comment]
Great! Feel free to merge anytime! :)

**Pull request merged**